### ISSUE TITLE
Unknown attribute usePrivateCurrentEnabled

### ISSUE THREAD

**Issue by Pyroil** [tillsteinbach/WeConnect-python#196]
I get the following errors when using the update() command:
/vehicles/XXXXXXXXXXXXXXXXXX/domains/automation/chargingProfiles/profiles/1/options: Unknown attribute usePrivateCurrentEnabled with value True
/vehicles/XXXXXXXXXXXXXXXXXX/domains/chargingProfiles/chargingProfilesStatus/profiles/1/options: Unknown attribute usePrivateCurrentEnabled with value True

I am using the latest version 0.60.3 on RPi
Using software version 3.2 of ID.4.

**Comment by ckarrie** [comment]
Can confim this:

```
[weconnect] /vehicles/XXXXXXXXXXXXXXXXXX/domains/automation/chargingProfiles/profiles/2/options: Unknown attribute usePrivateCurrentEnabled with value True
[weconnect] /vehicles/XXXXXXXXXXXXXXXXXX/domains/automation/chargingProfiles/profiles/1/options: Unknown attribute usePrivateCurrentEnabled with value True
[weconnect] /vehicles/XXXXXXXXXXXXXXXXXX/domains/chargingProfiles/chargingProfilesStatus/profiles/2/options: Unknown attribute usePrivateCurrentEnabled with value True
[weconnect] /vehicles/XXXXXXXXXXXXXXXXXX/domains/chargingProfiles/chargingProfilesStatus/profiles/1/options: Unknown attribute usePrivateCurrentEnabled with value True
```

**Comment by tillsteinbach** [comment]
fixed in 0.60.4
